src/index.html

```diff
@@ -1,12 +1,418 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="LogoMoyo"
+          class="header__logo"
+        >
+      </a>
+      <nav>
+        <ul class="header__nav">
+          <li class="header__nav-item">
+            <a href="#apple" class="header__nav-link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#samsung" class="header__nav-link">
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
```

Comment:
don`t forget to add empty line between sibling tags

Reply:
not fixed
check examples in the checklist
<img width="947" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/102411130/49ea41b4-2fe3-4f55-b3f9-7aa3863c8aa8">